### DISCUSSION TITLE
Auto replace installed profile during plugin install

### DISCUSSION THREAD

**Discussion by salimv** [elgatosf/elgato#82] [Help]
I use Streamdeck SDK's `switchToProfile` action to supoprt  switching between installed profiles. For context on why I created an action to switch profiles, please see [this discussion](https://github.com/orgs/elgatosf/discussions/27). My plugin ships with 6 installed profiles, one set for standard and another for XL device. 

When a customer re-installs or upgrades the plugin, the process ends up creating new copies of these installed profiles. Is there a way to also replace the profiles instead of creating copies every time the plugin is re-installed or upgraded? At the moment, customers have to manually delete each profile before installing the plugin. Alternatively, is there a way to remove installed profiles on plugin unintall?

**Comment by ChekTek** [comment]
Hi @salimv,

Unfortunately, it is not possible at this time, but we have a ticket in our backlog to investigate a better solution for this scenario
